### ISSUE TITLE
Add support for AWS EFA

### ISSUE THREAD

**Issue by threeseed** [bottlerocket-os/bottlerocket#1031]
**What I'd like:**

Built in package for the AWS Elastic Fabric Adapter driver.

**Comment by webern** [comment]
Thank you @nadenf for opening this tracker. We'll take a look at this soon.

**Comment by stmcginnis** [comment]
Just catching up on this - is this still needed? Can you clarify if a driver is needed? Or is it some sort of management CLI?

Is there some functionality missing right now?

**Comment by bryantbiggs** [comment]
Yes, this would greatly help with enabling ML and HPC workloads.
- Docs on installing EFA w/ NVIDIA drivers and device components https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa-start-nccl-base.html
- An example use case https://pages.awscloud.com/amazon-ec2-p4d.html - its not on ECS or EKS but its representative of the architecture and setup users are trying to achieve with ML and HPC workloads on containerized platforms

**Comment by tanvp112** [comment]
Hi @stmcginnis , may I know if bottlerocket support EFA already?

**Comment by stmcginnis** [comment]
I am not actively working on Bottlerocket anymore. @arnaldo2792 may be able to provide an update on EFA support.

**Comment by arnaldo2792** [comment]
@monirul has taken over this, we will post updates to this issue as we implement the features. 

(@stmcginnis :wave: )

**Comment by ytsssun** [comment]
We have launched EFA support starting from v1.28.0.

https://aws.amazon.com/about-aws/whats-new/2024/12/bottlerocket-elastic-fabric-adapter-ai-ml-workloads/
